### ISSUE TITLE
sorting doesn't work

### ISSUE THREAD

**Issue by plegall** [Piwigo/Piwigo-Mobile#90]
I don't think this feature is very useful (it is an "advanced feature" in my opinion). By default, sorting should be the same as sorting configured in Piwigo.

The "sort by Image ID" is not "understandable" (and doesn't work for me). I'm sure 99% of users think it sorts by the number in the file name (ie img_1234.jpg > img_1235.jpg), which is absolutely different.

In the list or sort criteria "Name Asc/Desc" should be "Title Asc/Desc" (see labels used in Piwigo).

Still in the list, I don't know what is "Date". In Piwigo we have 2 dates : "date created" and "date posted".

To sum up, I think the iOS app should rely on what was already done (and improved years over years) in Piwigo, ie take the same sorting list :+1: 

Default
Photo title, A → Z
Photo title, Z → A
Date created, new → old
Date created, old → new
Date posted, new → old
Date posted, old → new
Rating score, high → low
Rating score, low → high
Visits, high → low
Visits, low → high

**Comment by EddyLB** [comment]
Ho Pierrick,

I have fixed sort issues (mix between date_posted/available and date_creation).
I've also changed the sort criteria so that they are identical to those of the Web application.

See version 2.0.2 build 17 in TestFlight.
